_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __, localize } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiry_date: PropTypes.string,
```

Comment:
I don't want to let them be required, as I feel that in case of strange or invalid data the component should adjust instead of erroring out. We can make them all required if that's not the preferred approach, though.

Reply:
Setting a prop to be `isRequired` isn't a blocker for the component to work, it will just result in some React warnings in the browser consoles. I'm fine with not requiring them if it doesn't make sense for you.

Reply:
Tweaked a bit with 42e250c2a527d9eb7ad4b9c1140caa3106dcbaa4, but honestly people can pass a blank string and it'll work, technically?

Reply:
Technically yes. The idea with required props is to set expectations about what props the component needs in order to function. Of course we can always get around those limitations if we want 😉